cachito/web/status.py

```diff
@@ -37,8 +38,9 @@ def _service_ok(url):
     :param str url: url to fetch
     :return: tuple (ok: bool, reason: str or None)
     """
+    config = get_worker_config()
     try:
-        resp = no_retry_session.get(url)
+        resp = retry_session.get(url, timeout=config.cachito_api_timeout)
```

Comment:
The `cachito_api_timeout` is too long for this purpose. I would generally expect the status endpoint to respond in less than 10 seconds, for the individual requests around 3 seconds (or slightly larger than 3 seconds as recommended here https://docs.python-requests.org/en/master/user/advanced/#timeouts) should be enough.

I don't think this needs to be configurable, IMO a constant in status.py would do. But if you do want add a config option for this I won't stop you :-)

Reply:
@chmeliik Do you know why api timeout is located in the worker config file?

Reply:
@akhmanova it is the timeout that workers use when communicating with the API